client/src/features/rootV2/RootV2.tsx

```diff
@@ -140,8 +149,11 @@ function GroupsV2Routes() {
     <Routes>
       <Route
         path={RELATIVE_ROUTES.v2.groups.new}
-        element={<LazyGroupV2New />}
+        element={
+          <Navigate to={`${ABSOLUTE_ROUTES.v2.root}#createGroup`} replace />
```

Comment:
```suggestion
          <Navigate to={{ pathname: ABSOLUTE_ROUTES.v2.root, hash: "create-group" } replace />
```

Note: Ideally these would be handled as effects with `navigate()`.